pkg/process/logging.go

```diff
@@ -40,6 +41,12 @@ func newLogger() (*zap.Logger, error) {
 		levelEncoder = zapcore.CapitalLevelEncoder
 	}
 
+	winFileSink := func(u *url.URL) (zap.Sink, error) {
+		// Remove leading slash left by url.Parse()
+		return os.OpenFile(u.Path[1:], os.O_WRONLY|os.O_APPEND|os.O_CREATE, 0644)
```

Comment:
Would this panic when you pass in something like `winfile://`?

Reply:
yes, for `winfile://g:\test.log` -> `Error: couldn't open sink "winfile://g:\\test.log": can't parse "winfile://g:\\test.log" as a URL: parse winfile://g:\test.log: invalid port ":\\test.log" after host`

Reply:
I mean just `winfile://`?

Reply:
oh, `Error: couldn't open sink "winfile://": no sink found for scheme "winfile"`

Reply:
I was more worried that it will panic with an out-of-bounds.

Reply:
It looks it stops before entering `winFileSink()`. I pushed it as a draft because I didn't look too much into it, just to show idea.